index.html

```diff
@@ -422,7 +422,119 @@ <h1 class="contact_form_heading">
 
   </div>
   
+  <div class="modal_mob">
+    <div class="model_content">
+      <div class="pop_upHeader">
+        <div class="name" id="name">
+          <h1 class="nameOfProject">
+             
+          </h1>
+        </div>
+        <div class="pop-close">
+          <img class="close" src="images/popCancel.svg" alt="close button">
+        </div>
+      </div>
+
+      <div class="popUpInfo">
+       <div class="experience">
+        <ul class="popUpExp desk_var sub_tabs">
+          <li id="company"></li>
+          <li id="job"></li>
+          <li id="year"></li>
+        </ul>
+      </div>
+
+      <div class="pop_Image">
+        <img src="#" alt="#" id="img">
+      </div>
+
+      <div class="Popcontent">
+        <p id="pop_Text">
```

Comment:
- [ ] The popup follows the design for both desktop and mobile version from the chosen template in Figma (e.g., font, colors, images, tests, margins).
- Here is your pop-up modal.
- Here is the original popup modal